README.md

```diff
@@ -23,7 +23,7 @@ Any help is greatly appreciated, feel free to submit pull-requests or open issue
 
 ## Installation
 
-In **Rails 3 and 4**, add this to your Gemfile and run the `bundle install` command.
+In **Rails 3, 4 and 5 **, add this to your Gemfile and run the `bundle install` command.
```

Comment:
The docs now say that Cancancan supports only Rails >= 3. I think is not necessary to explicitly specify all the versions and not even Rails. Just Leave: `Add this to your Gemfile...`